identity-concentrator/cloudsql.tf

```diff
@@ -0,0 +1,21 @@
+module "cloudsql" {
+  source        = "github.com/broadinstitute/terraform-shared.git//terraform-modules/cloudsql-postgres?ref=cloudsql-postgres-1.0.0-tf-0.12"
```

Comment:
Is the version harded here, and we need to manually upgtade everthere whenever we need to?

Reply:
Yes, the version of the Postgres cloudSQL module is hard-coded. We try to lock versions of modules from the `terraform-shared` repo whenever possible because many teams use the modules from that repo so we want to enable everyone to iterate quickly but not break each other's stuff.